back-end/controllers/addImageToCollection.js

```diff
@@ -0,0 +1,24 @@
+const Sequelize = require("sequelize");
+const db = require("../db/dbconnection");
+const User = require("../db/models/user")(db, Sequelize);
+const Image = require("../db/models/image")(db, Sequelize);
+// imports, used for queries later 
+
+//getCollectionImages();
+//.forEach(img => res.json(img))
+const addImageToCollection = (req, res) => {
+    const { email } = req.body;
+    User.findOne({ where: { email: email} })
+    .then(user => {
+        if (!user) {
+            return res.status(422).json({error: 'The specified user does not exist'});
+        } else {
+           await User.addCollectionImages([Image])
```

Comment:
Close, just one thing.

You need to pass in an array of images from the front end.

so up in line 10 add
```
const { images } = req.body
```

and then here pass in images.

otherwise, looks good.